azurerm/provider.go

```diff
@@ -146,6 +146,7 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_container_service":                       resourceArmContainerService(),
 			"azurerm_container_group":                         resourceArmContainerGroup(),
 			"azurerm_cosmosdb_account":                        resourceArmCosmosDBAccount(),
+			"azurerm_data_factory_v2":                         resourceArmDataFactoryV2(),
```

Comment:
>			"azurerm_data_factory_v2": [](start = 0, length = 29)

V2 seems to be redundant information here, coz it's the current version and we never release V1 before. We can explain it in the documentation, but we don't need to add "_v2" here. That's my 2 cents.

Reply:
You may take a look at Azure portal, it allows end user to select v1 or v2. 

---
In reply to: [210760621](https://github.com/terraform-providers/terraform-provider-azurerm/pull/1773#discussion_r210760621) [](ancestors = 210760621)